pidfile/pidfile.go

```diff
@@ -19,6 +19,9 @@ func Create(pidfile string) error {
 	}
 	if pidString, err := ioutil.ReadFile(pidfile); err == nil {
 		if pid, err := strconv.Atoi(string(pidString)); err == nil {
+			if pid == os.Getpid() {
```

Comment:
I think it's better to leave some comments here to describe the background of this change.